lib/streamlit/elements/arrow_altair.py

```diff
@@ -759,9 +769,9 @@ def altair_chart(
         use_container_width: bool = False,
         theme: Literal["streamlit"] | None = "streamlit",
         # TODO(willhuang1997): This will need to be finalized to "rerun" / "ignore" or True / False
-        on_select: Union[str, Callable[..., None], True, False, None] = None,
+        on_select: Union[str, Callable[..., None], bool, None] = None,
```

Comment:
Why is this allowing `str`? To allow `rerun` and `ignore`? If so, it would be better to use Literals here.